components/emoji_picker/emoji_picker.test.js

```diff
@@ -0,0 +1,406 @@
+    it('Input search should be focus on mount', () => {
+      const searchInput = wrapper.find('.d-emoji-picker__search input');
+
+      expect(searchInput.element).toBe(document.activeElement);
+    });
+
+    it('Should focus the first available emoji when down keyarrow is pressed', async () => {
+      const searchInput = wrapper.find('.d-emoji-picker__search input');
+      const emoji = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__tab button');
+
+      await searchInput.trigger('keydown.down');
+
+      expect(document.activeElement).toBe(emoji.element);
+    });
+
+    it('Should jump through all the first emojis of each tab', async () => {
+      // Thumbs up emoji - Recently used
+      const firstRecentlyUsed = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(2) button');
+      // Skull and crossbones - People
+      const firstPeople = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(3) button');
+      // Sun - Nature
+      const firstNature = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(4) button');
+      // Coffe - Food
+      const firstFood = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(5) button');
+      // Soccer ball - Activity
+      const firstActivity = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(6) button');
+      // Anchor - Travel
+      const firstTravel = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(7) button');
+      // Keyboard - Objects
+      const firstObjects = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(8) button');
+      // Exclamation question mark - Symbols
+      const firstSymbols = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(9) button');
+      // White flag - Flags
+      const firstFlags = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(10) button');
+
+      await firstRecentlyUsed.trigger('keydown.tab');
+
+      expect(document.activeElement).toBe(firstPeople.element);
+
+      await firstPeople.trigger('keydown.tab');
+
+      expect(document.activeElement).toBe(firstNature.element);
+
+      await firstNature.trigger('keydown.tab');
+
+      expect(document.activeElement).toBe(firstFood.element);
+
+      await firstFood.trigger('keydown.tab');
+
+      expect(document.activeElement).toBe(firstActivity.element);
+
+      await firstActivity.trigger('keydown.tab');
+
+      expect(document.activeElement).toBe(firstTravel.element);
+
+      await firstTravel.trigger('keydown.tab');
+
+      expect(document.activeElement).toBe(firstObjects.element);
+
+      await firstObjects.trigger('keydown.tab');
+
+      expect(document.activeElement).toBe(firstSymbols.element);
+
+      await firstSymbols.trigger('keydown.tab');
+
+      expect(document.activeElement).toBe(firstFlags.element);
+    });
```

Comment:
each assertion here should be it's own test. This is probably a perfect case to use [test.each](https://vitest.dev/api/#test-each)

Reply:
Ye.. I remember tried to do it with it.each the thing is it is not cycle...
When you reach the last element (firstFlags.element) and you press `tab`, you move out the skin selector tabs.
Lemme see what I can do..

Reply:
Done, I just put a condition there.